.travis.yml

```diff
@@ -12,7 +12,7 @@ deploy:
     master: mittrainingplanner-master
     travis-test: mittrainingplanner-starmarek
     branch: mittrainingplanner-mbednar22
-    branch: mittrainingplanner-mkomorek
+    branch: mittrainingplanner-mkomorek project-structure-update
```

Comment:
Please look at previous apps. 
You have placed name of your branch incorrectly.
Should be : **project-structure-update: mittrainingplanner-mkomorek**

Reply:
Done